### PR TITLE
Completed prep-javascript-dom-creation exercise.

### PR DESCRIPTION
**Pull request by sarah-you**
prep-javascript-creation

### CODE REVIEW

**Comment by sarah-you** [comment]
Hi, 
Not sure why the h1 Hello World isn't showing up for me. I checked in console and it prints the Hello World there. I feel like I did something wrong in step 10. Could you let me know where I am making the mistake?

File: prep-javascript-dom-creation/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+</head>
+<body>
+  <div id="container">
+  </div>
+  <script>
+  const $container = document.querySelector('#container');
+  console.log($container);
+  const $heading = document.createElement('h1');
+  console.log($heading);
+  document.getElementsByTagName('$heading').textContent = 'Hello World!';
```
Comment (Zhiwutian) [inline prep-javascript-dom-creation/index.html:14]: you don't need to query here, you can use the `$heading` variable to update the textContent

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
make sure to reach out in slack if you have any questions!

**Comment by sarah-you** [comment]
prep-javascript-dom-creation 2

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**